tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -229,6 +229,21 @@ if (tensorflow_ENABLE_GPU)
   endif()
 endif()
 
+#  Resolve the installed version of Python (for Python.h and python).
+include(FindPythonInterp)
```

Comment:
Would it be enough to just add this line to `CMakeLists.txt`, and maybe add a test to see if `${PYTHON_EXECUTABLE}` has been set?

(It seems like it would be fine for the for `PYTHON_INCLUDE_DIR` to fail, and still build the non-Python parts of the repository.)